### ISSUE TITLE
FEAT: Snowflake Support

### ISSUE THREAD

**Issue by phillpafford** [beekeeper-studio/beekeeper-studio#443]
Snowflake DB has a NodeJS Driver https://docs.snowflake.com/en/user-guide/nodejs-driver.html

**Comment by rathboma** [comment]
Yes 100%. Just takes some time, and I don't have a snowflake install to test it against.

**Comment by matt-strautmann** [comment]
Hi, I am working mostly in Snowflake and would love to try your tool. I have access to a few Snowflake environments for testing. @rathboma let me know how I can help or test if you already have a test build created.

**Comment by fgiroud** [comment]
Would love to have snowflake too

**Comment by oliver-tunnard** [comment]
+1 for Snowflake

**Comment by windowman271** [comment]
any update?

**Comment by randypitcherii** [comment]
Hey! Just purchased an ultimate subscription and would love to have snowflake support.

Snowflake instances are free for a month at a time to set up and develop against. Just in case that helps.

Thanks!

**Comment by cagartner** [comment]
Any update? I bought the license just because your guys had the Snowflake support "Soon", but now checking here, I'm afraid that this could not be so soon :(

**Comment by johnoscott** [comment]
I too just purchased an Business license thinking "they must have Snowflake support by now" ~ when is it coming ?

**Comment by neilljordan** [comment]
Another vote for Snowflake support 👍

**Comment by KalebJS** [comment]
Same here-this is a blocker to any possibility of adoption at my company.

**Comment by briankski** [comment]
+1 vote for snowflake support

**Comment by not-night-but** [comment]
Just merged initial Snowflake support into master :) Will be coming in the next feature release (`v5.9`)

**Comment by rathboma** [comment]
This is out in v5.9 now!
